### PR TITLE
Use identifiers.js as single source of truth

### PR DESCRIPTION
**Pull request by jonandernovella**
**Related issue(s) and PR(s)**  
This PR contributes to #1038

Should be completed after merging https://github.com/MetabolicAtlas/data-generation/pull/36

**Type of change**  

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other
 
**List of changes made**  

- Updated build step to copy identifiers.js
- Use new object names in code

**Testing**  

- A certain branch in other repos is needed: **fix/unify-identifiers** from https://github.com/MetabolicAtlas/data-generation/pull/36

### CODE REVIEW

**Review by perjo — APPROVED** [review]
Looks good and tested together with https://github.com/MetabolicAtlas/data-generation/pull/36 without problems (should be merged together)

**Comment by mihai-sysbio** [comment]
As part of testing, please also check that links to cross-references still work as expected on GEM Browser and GotEnzymes pages.

**Comment by mihai-sysbio** [comment]
Let me try again: when testing the changes implemented in this PR, in addition to the usual review, please also test that everything works in the interface, especially in those pages that I know are richer in cross-references. If still unclear let's take it on Slack.
